### ISSUE TITLE
Syncany has delete 4/5 of my files after a new Computer was added to my Repo.

### ISSUE THREAD

**Issue by alkaphreak** [syncany/syncany#538]
Related to #533 

From my laptop, I've made a repo to sync on a remote ssh server through GUI, everything went fine.

Recently I've connected a desktop computer the same way with an empty repo to get everything that was up from my laptop, and the sync started with nearly 50Go. That was last week.

Yesterday I've seen that 9.6Go of data where download on my desktop, and today there were only the same files on my laptop.

Everything else has gone ... And obviously i deleted my laptop's backup to put them on my desktop and server ...

Help !

**Comment by pimotte** [comment]
Stop all the daemons running on the system. Please email us your logs from both the desktop and the laptop. (Both from ~/.config/syncany/logs and the folder (folder/.syncany/logs)) at team@syncany.org.

You should be able to restore specific files using the `syncany ls` and `syncany restore commands`

**Comment by mihaisecasiu** [comment]
Something similar happened to me. I lost a lot of files because of this. I usually sync using the gui but that is sometimes too slow because I have multiple folders so sometimes I use the sy command ( 0.4.7 on osx and linux ). I did sy up but it told me there are remote changes so when I did sy down it deleted a lot of files from the folder. I tried to restore with sy ls but I get nothing for the files that were deleted.

**Comment by pimotte** [comment]
I'm sorry this keeps happening.

@mihaisecasiu Could you send all the logs of any clients connected to that repo to the above address? All the logs up until this point have not contained information about the moment Syncany marked the files as deleted, and without further information on this we have little chance to actually fix this.

**Comment by alkaphreak** [comment]
I'm trying to take back and retrieve the maximum of data available and then i will send the logs. I'm a bit busy these days.

**Comment by mihaisecasiu** [comment]
I can't send the log files because of the sensitive information contained in the file names but I looked over and it seems that when comparing the files in the repository with the local ones it just detected most of them as being deleted in the repository so it just went on to delete the local files too. What's even worse is that there's no way to recover them. 
"syncany log" shows the list of all files that were deleted but "syncany ls" on any of those files show nothing.

One would think that syncany would only detect a file as deleted if the file was at some point in the remote repository otherwise a local file that's not in the remote repository should be considered a new file so it should be added to remote rather than removed from local. 
So maybe my local files were never submitted to the remote repository ( but then why was my folder reported as synchronized ). Or maybe the files were submitted but the mechanism for detecting them as present and the mechanism for listing and recovering them is broken.  

I have always had problems with synchronizing large folders, I even commented about this on other issues. Maybe the fact that I had multiple large folders is what broke it. 
I was also never able to run syncany continuously with about 4-5 relatively large folders ( 10-80k files, a few GB) on my macbook pro (16Gb ram) because it would become impossible to use anything on it anymore and files would still not completely sync.

I appreciate what you are trying to do but maybe the main focus should be on creating a simple system that just works and doesn't lose user's files before implementing new features. 

For now I have gave up and just moved to a paid service that so far seems to work well.

**Comment by alkaphreak** [comment]
Doing the same. I can't afford to lose my data or being unable to access it when i work remotely so i'm going back to dropbox.

I will follow the devlopment of Syncany and, will try again when the project had gain in maturity, it's still in alpha so it's not safe to use it for a daily tool. It's not sufficiently reliable.

**Comment by pimotte** [comment]
I have tried to reproduce this with my Arch Linux setup by syncing a reasonably large amount of data to a Windows VM, but have not managed to. We can't really move forward without more details. Any and all logs and information would help to resolve this.

**Comment by mihaisecasiu** [comment]
I can give you a few more details about my setup. 
My main computer now is a macbook pro with osx el capitan. I had 5 repositories setup, 3 of them had between 30k and 70k files in them. I used the sftp plugin with syncany 0.4.7. I had the same files on an ubuntu computer and synced them. The ubuntu sync is much faster than the osx one. The computers have the same amount of ram but the cpu is better on the ubuntu computer and the files are stored on an SSD there. In the beginning I used only the syncany gui to sync files and at some point in time both the mac and ubuntu computers had the "synchronized" icon in the tray, but as I mentioned it was making the mac too slow so I stopped it. I chose to just use sy up and sy down to sync whenever I had changed something important enough to sync. 
So I changed something on the mac, did "sy up" but it told me there were remote changes so when I did "sy down" it just deleted most of the files in that directory and because the other computer was also on and syncany gui was started on it, the deletion was "copied" to the other computer so basically I lost most of the files.
Why were those files marked as deleted by remote in the repository when none of the computers deleted those files. And if they were marked as deleted in the repository then how could it happen that sy ls doesn't show the previous versions of a file while sy log shows the deletion.
